drivers/video/video_common.c

```diff
@@ -82,6 +82,8 @@ void video_buffer_release(struct video_buffer *vbuf)
 	struct mem_block *block = NULL;
 	int i;
 
+	__ASSERT(vbuf, "Invalid arguments");
```

Comment:
Do not use implicit pointer to boolean conversion, this is a required rule, see https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html rule 85.

```suggestion
	__ASSERT(vbuf != NULL, "Invalid arguments");
```